### DISCUSSION TITLE
Scaling Simulations in Simmer

### DISCUSSION THREAD

**Discussion by swaheera** [r-simmer/simmer#319] [General]
Suppose I have the following parameters:

- Arrival rate (λ) = 300
- Number of servers (c) = 2
- Service rate per server (μ) = 5.1

This is taking a long time in simmer since the individual arrivals are actually being simulated. 

I was wondering if it is possible to scale initial parameters down, simulate to get a faster simulation - and then scale the results back up?

E.g. Scaled system (divide by 100):

Arrival rate (λ') = 3
Number of servers (c') = 2 (unchanged)
Service rate per server (μ') = 0.051

Is this a valid approach within Simmer?

Thank you!

**Comment by Enchufa2** [comment]
I don't recommend scaling anything down, because 1) it's easier to make mistakes, and 2) if you don't make any and the system is equivalent, then you'll end up needing to generate a similar number of events and the simulation will take the same time. There's no "free beer" here. What you can do is to 1) set the minimum simulation time required to get proper results, so that one simulation is as fast as it can get, and 2) parallelize different replications. There's documentation about the latter. Using `future_lapply` and similar functions makes it super-easy.
